packages/app-page-builder/src/pageEditor/helpers.ts

```diff
@@ -9,7 +9,7 @@ import {
     PbEditorPageElementVariableRendererPlugin
 } from "~/types";
 
-export const createBlockReference = (name: string): PbEditorElement => {
+export const createBlockReference = (name?: string): PbEditorElement => {
```

Comment:
`name` can really be `undefined`?

Reply:
Plugin name property can be `undefined` as defined [here](https://github.com/webiny/webiny-js/blob/next/packages/plugins/src/types.ts#L5). Also `plugins.byName` function accepts `undefined`, so it should not introduce any issues.